x-pack/plugins/security_solution/common/endpoint/policy/migrations/to_v7_12_0.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectMigrationFn, SavedObjectUnsanitizedDoc } from 'kibana/server';
+import { cloneDeep } from 'lodash';
+import { PackagePolicy } from '../../../../../fleet/common';
+import { ProtectionModes } from '../../types';
+
+export const migratePackagePolicyToV7120: SavedObjectMigrationFn<PackagePolicy, PackagePolicy> = (
```

Comment:
Here upgrading the package policy will not trigger a policy change to the agent. It is the wanted feature right?

Reply:
yes, after talking to people on the team, it is expected behavior